src/Components/Notifications.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { ErrorTypes } from '../types/ErrorTypes';
+
+type Props = {
+  setError: (error: ErrorTypes | null) => void,
+  error: ErrorTypes | null,
+};
+
+export const Notifications: React.FC<Props> = ({ setError, error }) => {
+  const [isHidden, setIsHidden] = useState(false);
+
+  const deleteError = () => {
+    setIsHidden(true);
+    setTimeout(() => {
+      setError(null);
+    }, 1000);
```

Comment:
do you really need the setTimeout here? maybe it would be better to remove the error immediately?
```suggestion
    setError(null);
```